splunk/Dockerfile

```diff
@@ -30,6 +31,7 @@ RUN mkdir -p /opt/splunk/var/lib/splunk
 COPY --chmod=777 --from=builder /opt/splunk/etc/apps/search/local/indexes.conf /opt/splunk/etc/apps/search/local/indexes.conf
 COPY --chmod=777 --from=builder /var/splunk_buildtime_db/splunk/ /opt/splunk/var/lib/splunk/
 
+
```

Comment:
Nitpick, but as it has dragged for long, let's minimize the changes of this PR to what is absolutely necessary.